guides/v2.1/coding-standards/docblock-standard-javascript.md

```diff
@@ -274,7 +276,7 @@ Identifies the type of a variable, property, or expression.
 var&nbsp;hexId&nbsp;=&nbsp;hexId;
 </pre>
 
-<h4 id="frontend-dev-guide-javascript-js-coding-docblock-typedef">@typedef</h4>
+#### @typedef
```

Comment:
Please append `{#frontend-dev-guide-javascript-js-coding-docblock-typedef}` to preserve the custom id.